app/src/main/java/atlant/moviesapp/fragments/MovieFavoritesFragment.java

```diff
@@ -0,0 +1,235 @@
+                        c.drawText("Remove",(float) itemView.getLeft() + width,(float) itemView.getTop() + width,t);
+                    } else {
+                        p.setColor(getResources().getColor(R.color.standardYellow));
+                        t.setColor(Color.parseColor("#FFFFFF"));
+                        RectF background = new RectF((float) itemView.getRight() + dX, (float) itemView.getTop(),(float) itemView.getRight(), (float) itemView.getBottom());
+                        c.drawRect(background,p);
+                        RectF icon_dest = new RectF((float) itemView.getRight() - 2*width ,(float) itemView.getTop() + width,(float) itemView.getRight() - width,(float)itemView.getBottom() - width);
+                        c.drawText("Remove",(float) itemView.getLeft() + width,(float) itemView.getTop() + width,t);
+
+                    }
+                }
+                super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive);
+            }
+        };
+
+        ItemTouchHelper itemTouchHelper = new ItemTouchHelper(simpleItemTouchCallback);
+        itemTouchHelper.attachToRecyclerView(recyclerView);
+
+        return v;
+    }
+
+    @Override
+    public void showError() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setMessage("Error loading data, please check your connection")
+                .setTitle("Error");
+
+        builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int id) {
+                // User clicked OK button
+            }
+        });
+        AlertDialog dialog = builder.create();
+        dialog.show();
+
+    }
+
+    @Override
+    public void showMovies(List<Movie> data) {
+        if (!(data.size() > 0)) {
+            adapter.setMoreDataAvailable(false);
+        } else {
+            favoriteMovies.addAll(data);
+            adapter.notifyDataChanged();
+        }
+
+    }
+
+    @Override
+    public void showTvShows(List<TvShow> data) {
+
+    }
+
+    @Override
+    public void showProgress() {
+        progressBar.setVisibility(View.VISIBLE);
+
+    }
+
+    @Override
+    public void hideProgress() {
+        progressBar.setVisibility(View.INVISIBLE);
+
+    }
+
+    @Override
+    public void onStop() {
+        super.onStop();
+        if (presenter != null)
+            presenter.onStop();
+    }
+
+    @Override
+    public void onDestroy() {
+        if (presenter != null)
+            presenter.onDestroy();
+        super.onDestroy();
+    }
+
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        if (!resumeHasRun) {
+            resumeHasRun = true;
+            return;
+        }
+        adapter.clear();
+        favoriteMovies.clear();
+        presenter.getMovieFavorites(1);
```

Comment:
currentPage as the parameter to be sent in the presenter.getMovieFavourites(currentPage);